### PR TITLE
fix: configuration file link broken

### LINKED ISSUE #1809: doc: link is broken in deploy-with-docker
## Issue description
https://apisix.apache.org/docs/api/conf/conf.yaml is broken  in deploy-with-docker

### PR DESCRIPTION
**Pull request by stu01509**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [X] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**
Fix configuration file link broken

**Related issues**

fix/resolve #1809 

**Checklist:**

- [ ] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Review by starsz — APPROVED** [review]
LGTM

**Review by nic-chen — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

File: docs/en/latest/deploy-with-docker.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ $ docker build -t apisix-dashboard:$tag . --build-arg APISIX_DASHBOARD_VERSION=m
 
 1. Preparing configuration files
 
-Before starting the container, the configuration file `conf.yaml` needs to be prepared inside the **host** to override the default [configuration file](../api/conf/conf.yaml) inside the container.
+Before starting the container, the configuration file `conf.yaml` needs to be prepared inside the **host** to override the default [configuration file](https://github.com/apache/apisix-dashboard/blob/master/api/conf/conf.yaml) inside the container.
```
Comment (juzhiyuan) [inline docs/en/latest/deploy-with-docker.md:55]: 🤔 For versioned package..

Comment (stu01509) [reply docs/en/latest/deploy-with-docker.md:55]: Should I change to the relative path?

Comment (juzhiyuan) [reply docs/en/latest/deploy-with-docker.md:55]: ok for me

**Review by iamayushdas — APPROVED** [review]

**Comment by juzhiyuan** [comment]
CI pending..

**Review by KishaniKandasamy — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Comment by imjoey** [comment]
@stu01509 @juzhiyuan The CI seems pending for quite a long time. I'm a little confused about why the workflow `Frontend E2E Test` was triggered since the `path-ignores` configuration was fine to me. I'm afraid that there are several PRs are already delayed by the same problem. Any insights on this? Thanks for the help.

**Comment by imjoey** [comment]
> @stu01509 @juzhiyuan @LiteSun The CI seems pending for quite a long time. I'm a little confused about why the workflow `Frontend E2E Test` was triggered since the `path-ignores` configuration was fine to me. I'm afraid that there are several PRs are already delayed due to the same problem. Any insights on this? Thanks for the help.

The `Frontend e2e test` is always stuck, I filed a ticket to the Infra team to check if it has been set as `Required`. Put ticket https://issues.apache.org/jira/browse/INFRA-21906 here for tracking.

**Comment by juzhiyuan** [comment]
> > @stu01509 @juzhiyuan @LiteSun The CI seems pending for quite a long time. I'm a little confused about why the workflow `Frontend E2E Test` was triggered since the `path-ignores` configuration was fine to me. I'm afraid that there are several PRs are already delayed due to the same problem. Any insights on this? Thanks for the help.
> 
> The `Frontend e2e test` is always stuck, I filed a ticket to the Infra team to check if it has been set as `Required`. Put ticket https://issues.apache.org/jira/browse/INFRA-21906 here for tracking.

Thanks. But it's wired because Backend CI should be required too 🤔

**Comment by imjoey** [comment]
> > > @stu01509 @juzhiyuan @LiteSun The CI seems pending for quite a long time. I'm a little confused about why the workflow `Frontend E2E Test` was triggered since the `path-ignores` configuration was fine to me. I'm afraid that there are several PRs are already delayed due to the same problem. Any insights on this? Thanks for the help.
> > 
> > 
> > The `Frontend e2e test` is always stuck, I filed a ticket to the Infra team to check if it has been set as `Required`. Put ticket https://issues.apache.org/jira/browse/INFRA-21906 here for tracking.
> 
> Thanks. But it's wired because Backend CI should be required too 🤔

This CI issue has already fixed by Apache Infra team, see https://issues.apache.org/jira/browse/INFRA-21906 for detail. We can merge this now. Thanks.

**Pull request merged**